syft/frameworks/torch/hook.py

```diff
@@ -187,11 +185,14 @@ def _which_methods_should_we_auto_overload(self, tensor_type):
 
         to_overload = []
 
-        for attr in dir(tensor_type):
+        for attr in dir(TorchTensor):
```

Comment:
I don't think this change is correct. Not all tensor types have the same attributes as torch tensor

Reply:
resolved in ddf8533726553bf7e53b95281d9013e402b7b546